packages/amplication-data-service-generator/src/server/resource/dto/graphql/entity-list-relation-filter/create-entity-list-relation.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { print } from "recast";
+import { Entity } from "../../../../../types";
+import {
+  EXAMPLE_ID_FIELD,
+  EXAMPLE_SINGLE_LINE_TEXT_FIELD,
+} from "../../../util/test-data";
+import { createWhereInput } from "../../create-where-input";
+import {
+  createEntityListRelationFilter,
+  createEntityListRelationFilterID,
+} from "./create-entity-list-relation-filter";
+
+const EXAMPLE_ENTITY: Entity = {
+  id: "EXAMPLE_ENTITY_ID",
+  name: "ExampleEntity",
+  displayName: "Example Entity",
+  pluralDisplayName: "Example Entities",
+  fields: [EXAMPLE_ID_FIELD, EXAMPLE_SINGLE_LINE_TEXT_FIELD],
+  permissions: [],
+};
+const EXAMPLE_WHERE_INPUT = createWhereInput(EXAMPLE_ENTITY);
+
+describe("createEntityListRelation", () => {
+  test("create list relation", async () => {
+    expect(
+      print(
+        await createEntityListRelationFilter(
+          EXAMPLE_ENTITY,
+          EXAMPLE_WHERE_INPUT
+        )
+      ).code
+    ).toEqual(`@InputType()
```

Comment:
Way not to use snapshot inline test instead?

Reply:
https://jestjs.io/docs/snapshot-testing#inline-snapshots

Reply:
I saw all the other Input has test files, so I added one too

Reply:
Yes, all the other inputs have a test that tests there AST object.
You made a good test (my preferred way ;) ) that checked the generation of the dto, i just suggest you a better way (in my opinion) to do a string compressing test, a snapshot test

Reply:
I am not sure why they say on the docs:
"Snapshot tests are a very useful tool whenever you want to make sure your UI does not change unexpectedly."
A template creates EntityListRelationFilter, and it has no reason to change unless someone changes the template.
Correct me if I am wrong

Reply:
I changed it to snapshot test. Thanks :)